docs/markdown/Getting-meson_ptbr.md

```diff
@@ -8,37 +8,37 @@ pacotes, você precisa baixa-lo da [página inicial do Python]. Veja abaixo
 
 ## Baixando o Meson
 
-*Releases* do Meson podem ser baixadas da [página de releases do GitHub]
+*Releases* do Meson podem set baixadas da [página de releases do GitHub]
```

Comment:
This is not English and should not be checked as such.

Reply:
(I'm not even going to try to review this file, yeah...)